src/lib/monitor-adapter.js

```diff
@@ -19,12 +21,15 @@ const isUndefined = a => typeof a === 'undefined';
  * @param {number} y - The monitor y position
  * @return {object} The adapted monitor with label and category
  */
-export default function ({id, index, opcode, params, value, x, y}) {
+export default function ({id, spriteName, index, opcode, params, value, x, y}) {
```

Comment:
Can we separate the monitor sprite name stuff into a separate PR? It depends on some VM changes, while the other parts of this do not.

Reply:
Broken out at https://github.com/LLK/scratch-gui/pull/899